In [5]:
alphabet = "bcdfghjklmpqruvwxyz" #removing most commonly used alphabets: e, t, a, i, n, o, s
score = 0
names = input("Enter your name give it a space and type in your friend's name: ").lower()
for character in names:
    if character in 'aeiou':
        score += 5
    if character in 'friends':
        score += 10
    if character in alphabet:
        score += alphabet.find(character)
    else:
        score += 0
if score > 100:
    print("Your friendship sore is: ", score)
    print("Congratulation! you both are best friends")
else:
    print("Your friendship score is: ", score)

Enter your name give it a space and type in your friend's name: Ramayana Mahabharat
Your friendship sore is:  139
Congratulation! you both are best friends


## Run 1:
Enter your name give it a space and type in your friend's name: Python java<br>
Your friendship score is:  77

## Run 2:
Enter your name give it a space and type in your friend's name: Data Pirates<br>
Your friendship score is:  99

## Run 3:
Enter your name give it a space and type in your friend's name: walter paige<br>
Your friendship sore is:  114
Congratulation! you both are best friends

## Run 4:
Enter your name give it a space and type in your friend's name: Ramayana Mahabharat<br>
Your friendship sore is:  139
Congratulation! you both are best friends

-------------------------------------------------------------------------------------------------------------------------------